flake8_cached/cli.py

```diff
@@ -38,8 +38,8 @@ def run_checks(self):
 def main():
     from flake8.main import cli
 
-    cli.main()
+    return cli.main()
 
 
 if __name__ == "__main__":
-    main()
+    raise SystemExit(main())
```

Comment:
It needs to raise only when the status code is not zero right?

Reply:
It also works when the status code is zero.
Actually, I have reused the way it is implemented in Flake8 😄

Reply:
it is better to wrap it in `sys.exit(main())`

Reply:
OK, I have updated the PR.